Source/GameMapEx.h

```diff
@@ -1,8 +1,6 @@
-// GameMapEx.h
-// Extensions to the GameMap class
-#ifndef _GAMEMAPEX_H_
-#define _GAMEMAPEX_H_
+#pragma once
 
+// Extensions to the GameMap class
 class GameMapEx : public GameMap

```

Comment:
This file depends on Outpost2DLL I think to bring in the definition of GameMap. This isn't realized until all the projects are brought in via OP2MissionSDK.

I think to be more proper, we should add an include for Outpost2DLL.h to this file. Maybe we can do that without actually adding Outpost2DLL as a submodule to this project since it is assumed one is pulling down the entire OP2MissionSDK to use HFL? Looking for @DanRStevens' thought on this...

Reply:
Heh, that is a bit of an ugly problem.

I'm ok with ignoring it for now.